server/src/main/java/fr/uga/l3miage/tp4/models/ExamEntity.java

```diff
@@ -0,0 +1,27 @@
+package fr.uga.l3miage.tp4.models;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Data
+@Entity
+public class ExamEntity {
+    @Id
+    private long id;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    @Column(unique = true) private String name;
+
+    @ManyToOne()
+    private EcosSessionEntity ecosSessionEntity;
+
+    @ManyToMany(mappedBy = "examEntities")
+    private Set<SkillEntity> skillEntities;
+
```

Comment:
elle est unidirectionnelle 
```suggestion
    @ManyToMany
    private Set<SkillEntity> skillEntities;

```